sdk/lib/atl/atlimage.h

```diff
@@ -130,9 +142,8 @@ class CImage
         bf.SourceConstantAlpha = bSrcAlpha;
         bf.AlphaFormat = AC_SRC_ALPHA;
 
-        GetDC();
         BOOL ret = ::AlphaBlend(hDestDC, xDest, yDest, nDestWidth, nDestHeight,
-                                m_hDC, xSrc, ySrc, nSrcWidth, nSrcHeight, bf);
+                                GetDC(), xSrc, ySrc, nSrcWidth, nSrcHeight, bf);
```

Comment:
Why isn't the original code OK? Is `m_hDC` inaccessible now?

Reply:
This change can reduce lines.

Reply:
And makes code less obvious. Because before, whatever was using the HDC was between a `GetDC(); .... ReleaseDC();`. Now you have to find where the GetDC() is inside the code block before the `ReleaseDC()`. And in case you need to use more than once the hDC, what would you do? Call GetDC() more than once?
```c
some_function(params, GetDC(), other_params);
some_other_function(more_params, GetDC(), more_other_params);
ReleaseDC();
```
(of course not because you would get a reference count mismatch). Or insert an intermediate ReleaseDC() call just in-between? (not efficient).

Reply:
Reverted around `GetDC()`.